### DISCUSSION TITLE
Can we use Supabase using Blazor Javascript Interop

### DISCUSSION THREAD

**Discussion by sam-claire** [supabase/supabase#7516] [General]
Is this possible that we can use Blazor Javascript Interop in supabase?
Calling this code:

const { user, session, error } = await supabase.auth.signUp({
  email: 'example@email.com',
  password: 'example-password',
})

Thanks

**Comment by burggraf** [comment]
What happens when you try to execute that code?  (You might also check out our c-sharp community to see if anyone has any input on this:  [supabase-csharp](https://github.com/supabase-community/supabase-csharp))
